integration-tests/flake.nix

```diff
@@ -0,0 +1,92 @@
+{
+  description = "Integration tests";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+
+    # define gomod2nix which is a dependency to build Go lang plugins
+    gomod2nix = {
+      url = "github:tweag/gomod2nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+      inputs.utils.follows = "flake-utils";
+    };
+
+    # define chainlink-dev flake as a input
+    chainlink-dev.url = "git+ssh://git@github.com/smartcontractkit/chainlink-project-nix-poc?ref=main&dir=chainlink-dev"; # this could also be a project URL
```

Comment:
This import should point to a renamed repo `github.com/smartcontractkit/chainlink-dev` once the PoC is stabilised - @willyrgf do we have a ticket to track this?